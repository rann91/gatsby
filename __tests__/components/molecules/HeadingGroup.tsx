import { render } from '@testing-library/react'
import React from 'react'
import HeadingGroup from '../../../src/components/molecules/HeadingGroup'

describe('HeadingGroup', () => {
  it('renders correctly', () => {
    const { container } = render(
      <HeadingGroup title="Title" subtitle="Subtitle" />
    )

    expect(container).toMatchSnapshot()
  })
})
