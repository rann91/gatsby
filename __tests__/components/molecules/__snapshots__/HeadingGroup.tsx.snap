// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeadingGroup renders correctly 1`] = `
<div>
  <div
    class="css-1mpaekt-HeadingGroup"
  >
    <div
      class="css-146vepc-HeadingGroup"
      data-testid="subtitle"
    >
      Subtitle
    </div>
    <h2
      class="css-16nj6cn-HeadingGroup"
      data-testid="title"
    >
      Title
    </h2>
  </div>
</div>
`;
